src/applications/vaos/appointment-list/components/AppointmentsPage/AppointmentsPageV2.jsx

```diff
@@ -0,0 +1,279 @@
+    () => {
+      if (
+        !cancelInfo.showCancelModal &&
+        cancelInfo.cancelAppointmentStatus === FETCH_STATUS.succeeded
+      ) {
+        scrollAndFocus();
+      }
+    },
+    [cancelInfo.showCancelModal, cancelInfo.cancelAppointmentStatus],
+  );
+  const history = useHistory();
+
+  const isLoading =
+    pendingStatus === FETCH_STATUS.loading ||
+    expressCare.windowsStatus === FETCH_STATUS.loading ||
+    pendingStatus === FETCH_STATUS.notStarted ||
+    expressCare.windowsStatus === FETCH_STATUS.notStarted;
+
+  // TODO - Rename to 'AppointmentList' since the list now displays all appointment types.
+  const routes = (
+    <Switch>
+      <Route exact path="/" component={FutureAppointmentsListV2} />
+      <Route path="/upcoming" component={FutureAppointmentsListV2} />
+      <Route path="/requested" component={FutureAppointmentsListV2} />
+      <Route path="/past" component={PastAppointmentsList} />
+      <Route path="/cancelled" component={FutureAppointmentsListV2} />
+    </Switch>
+  );
+
+  function onChange(e) {
+    if (e.currentTarget.value === 'upcoming') {
+      history.push('/upcoming');
+    } else if (e.currentTarget.value === 'requested') {
+      history.push('/requested');
+    } else if (e.currentTarget.value === 'past') {
+      // const dateRangeOptions = getPastAppointmentDateRangeOptions();
+      // const selectedDateRange = dateRangeOptions[0];
+      // fetchPastAppointments(
+      //   selectedDateRange.startDate,
+      //   selectedDateRange.endDate,
+      //   0,
+      // );
+      history.push('/past');
+    } else if (e.currentTarget.value === 'cancelled') {
+      history.push('/cancelled');
+    }
+  }
+
+  return (
+    <PageLayout>
+      <h1 className="vads-u-flex--1">{pageTitle}</h1>
+      <DowntimeNotification
+        appTitle="VA online scheduling tool"
+        isReady
+        dependencies={[externalServices.vaosWarning]}
+        render={(props, childContent) => (
+          <WarningNotification {...props}>{childContent}</WarningNotification>
+        )}
+      />
+
+      {showScheduleButton && (
+        <ScheduleNewAppointment
+          isCernerOnlyPatient={isCernerOnlyPatient}
+          showCommunityCare={showCommunityCare}
+          showDirectScheduling={showDirectScheduling}
+          showHomePageRefresh={showHomePageRefresh}
+          startNewAppointmentFlow={() => {
+            recordEvent({
+              event: `${GA_PREFIX}-schedule-appointment-button-clicked`,
+            });
+            startNewAppointmentFlow();
+          }}
+        />
+      )}
+
+      {showCheetahScheduleButton && (
+        <ScheduleNewProjectCheetah
+          showHomePageRefresh={showHomePageRefresh}
+          startNewAppointmentFlow={() => {
+            recordEvent({
+              event: `${GA_PREFIX}-schedule-project-cheetah-button-clicked`,
+            });
+            startNewAppointmentFlow();
+          }}
+        />
+      )}
+
+      {!expressCare.enabled && (
```

Comment:
I would just remove this block, since it'll render tabs and I think we can assume we'll be showing express care requests

Reply:
Done.